src/gui/base/Checkbox.ts

```diff
@@ -82,7 +75,7 @@ export class Checkbox implements Component<CheckboxAttrs> {
 							cursor: a.disabled ? "default" : "pointer",
 							font: "inherit",
 							"background-color": theme.content_accent,
-							"mask-image": `url("data:image/svg+xml,${icon}")`,
+							"mask-image": `url("${Checkbox.getIcon(a.checked)}")`,
```

Comment:
can we do this once for each state instead of on every redraw?

Reply:
Good idea, I'll put a commit through.